### PR TITLE
demo/improve-a11y-btc

### PR DESCRIPTION
**Pull request by DJTechnoo**

### CODE REVIEW

**Review by marvin19 — CHANGES_REQUESTED** [review]

File: samples/stock/demo/area/demo.js [hunk 1]
```diff
@@ -154,7 +156,7 @@ Highcharts.setOptions({
 });
 
 // Configuring the chart.
-Highcharts.stockChart('container', {
+const chart = Highcharts.stockChart('container', {
```
Comment (kamil-musialowski) [inline samples/stock/demo/area/demo.js:159]: I believe we don't need the chart object in this demo 🤔

**Review by kamil-musialowski — CHANGES_REQUESTED** [review]
Everything looks good imo! Just one tiny leftover :)

**Review by kamil-musialowski — APPROVED** [review]

**Review by pawellysy — COMMENTED** [review]
LGTM!

**Review by pawellysy — APPROVED** [review]

**Review by pawelfus — APPROVED** [review]
Thanks!

**Review by bm64 — APPROVED** [review]

**Review by TorsteinHonsi — APPROVED** [review]
Thanks!

**Pull request merged**